python/lsst/meas/extensions/psfex/psfexStarSelector.py

```diff
@@ -121,6 +100,17 @@ def validate(self):
         if self.minFwhm > self.maxFwhm:
             raise pexConfig.FieldValidationError("minFwhm (%f) > maxFwhm (%f)" % (self.minFwhm, self.maxFwhm))
 
+    def setDefaults(self):
+        self.badFlags = [
+            "base_PixelFlags_flag_edge",
+            "base_PixelFlags_flag_saturated.center",
+            "base_PixelFlags_flag_cr.center",
+            "base_PixelFlags_flag_bad",
+            "base_PixelFlags_flag_suspect.center",
```

Comment:
Great catch. Also, I don't think we even have flag_suspectCenter (though maybe that has been ported since I last recorded schema).